code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -606,6 +618,16 @@ trans_to_atmos(var/datum/gas_mixture/target, var/amount=1, var/multiplier=1, var
 		for(var/datum/reagent/R in reagent_list)
 			R.reaction_dropper_obj(A, R.volume+volume_modifier)
 
+/datum/reagents/proc/get_equalized_temperature(temperature_A, thermalmass_A, temperature_B, thermalmass_B)
+	//Gets the equalized temperature of two thermal masses
+	if(temperature_A == temperature_B)
```

Comment:
There's really no point to this check. Directly checking equality of floating point values almost always returns false anyway due to minute differences.